desktop/src/main.rs

```diff
@@ -28,15 +28,78 @@ use winit::event::{ElementState, MouseButton, MouseScrollDelta, WindowEvent};
 use winit::event_loop::{ControlFlow, EventLoop};
 use winit::window::{Icon, WindowBuilder};
 
+#[derive(Clap, PartialEq, Debug)]
+pub enum GraphicsBackend {
+    Default,
+    Vulkan,
+    Metal,
+    Dx12,
+    Dx11,
+}
```

Comment:
Is it worth adding GL here as well?

Reply:
I thought about it but it'd just immediately crash. I think we should add it when it's actually supported.